api/libproject/devstart.rb

```diff
@@ -1987,21 +1999,23 @@ def create_terra_method_snapshot(cmd_name, *args)
     "--all-projects [all-projects]",
     ->(opts, _) { opts.all_projects = true},
     "Create snapshot in every AoU environment. Cannot be used with --project.")
-  op.opts.source_git_repo = "broadinstitute/gatk"
```

Comment:
What was the bug you hit with defaults? I can't imagine a situation where we actually need to specify an alternate value here, so this just seems like extra stuff to copy-paste to me.

Reply:
the `opts.___` field was being set to a different variable name so the tool would use the default "broadinstitute/gatk" even though a repo was being provided with the arguments.

In practice, I've always been specifying all the arguments. It is a little more to type but I like being able to verify that I'm specifying the right file.